tracing-attributes/src/lib.rs

```diff
@@ -579,3 +663,80 @@ fn instrument_err(args: &[NestedMeta]) -> bool {
         _ => false,
     })
 }
+
+// terrible machinery to make async_trait-like cases works
+// (following the approach suggested in
+// https://github.com/dtolnay/async-trait/issues/45#issuecomment-571245673)
+fn is_async_trait(block: &Block, block_is_async: bool) -> Option<String> {
```

Comment:
Nit, take it or leave it: In general, I try to reserve `is_` for functions that return a boolean. This is returning the _name_ of the `async-trait`-generated inner function, if it exists...